parts/k8s/cloud-init/artifacts/cse_main.sh

```diff
@@ -178,7 +181,9 @@ mkdir -p /opt/azure/containers && touch /opt/azure/containers/provision.complete
 ps auxfww > /opt/azure/provision-ps.log &
 
 if $FULL_INSTALL_REQUIRED; then
-  applyCIS || exit $ERR_CIS_HARDENING_ERROR
+  if [[ $OS != $COREOS_OS_NAME ]]; then
```

Comment:
let's change to `if [[ $OS == $UBUNTU_OS_NAME ]] && [ "$FULL_INSTALL_REQUIRED" = "true" ]; then` to match the above.

Reply:
That messes up our else, I can change the inner block to if ubuntu, tho